project/profiles/capacity4more/modules/c4m/field/c4m_field_og/c4m_field_og.module

```diff
@@ -119,7 +119,10 @@ function c4m_field_og_field_access($op, $field, $entity_type, $entity, $account)
  * Hides certain status values at form, according to users access rights.
  */
 function c4m_field_og_form_group_node_form_alter(&$form, &$form_state, $form_id) {
-  _c4m_field_og_set_status_flow($form, $form_state, 'group');
+  // If editing node, set the status flow.
+  if (isset($form['nid']['#value'])) {
```

Comment:
Status flow sets allowed values for 'c4m_og_status' field, as described at following diagram.
The problem here is that the action is supposed to execute on edit, and the alter is triggered on create, as well.

Don't think that group content type is related here. Why do you ask about it?